ingest/src/server_mobile.rs

```diff
@@ -484,6 +548,17 @@ pub async fn grpc_server(settings: &Settings) -> Result<()> {
         .create()
         .await?;
 
+    let (subscriber_mapping_event_sink, subscriber_mapping_event_server) =
+        file_sink::FileSinkBuilder::new(
+            FileType::VerifiedSubscriberMappingEvent,
```

Comment:
The filetype written to this sink should be the ingest report ( the wrapper around the original request )